qiskit/circuit/library/standard_gates/u3.py

```diff
@@ -186,6 +186,22 @@ class CU3Gate(ControlledGate):
     It is restricted to 3 parameters, and so cannot cover generic two-qubit
     controlled gates).
 
+    .. warning::
+
+       This gate is deprecated. Instead, the following replacement should be used
+
+       .. math::
+
+           CU3(\theta, \phi, \lambda) = CU(\theta, \phi, \lambda, 0)
+
+       .. code-block:: python
+
+          circuit = QuantumCircuit(2)
+          circuit.cu(theta, phi, lambda, 0, 1)
```

Comment:
This misses the 4th parameter of `CUGate` I think, no?
```suggestion
          gamma = 0
          circuit.cu(theta, phi, lambda, gamma, 0, 1)
```